sdk/python/kfp/cli/cli_test.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+import sys
+import unittest
+from unittest import mock
+
+from click import testing
+
+# Docker is an optional install, but we need the import to succeed for tests,
```

Comment:
In that case, `docker` should be test dependency, maybe add it to requirements-test.txt? WDYT?

Reply:
Good idea. I did two things:
1) I made the intent of the mock more clear with 288c944.
2) I added `docker` (and `mock`, which had a TODO note) to the `requirements-test.txt` file with 
63726ef. I chose not to pin the versions because we don't pin the versions in the other places these packages are/were listed (`presubmit-tests-sdk.sh` for `mock` and `setup.py` for `docker`.)